src/test/java/htsjdk/samtools/util/LogTest.java

```diff
@@ -0,0 +1,31 @@
+package htsjdk.samtools.util;
+
+import htsjdk.HtsjdkTest;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.util.List;
+
+public class LogTest extends HtsjdkTest {
+
+    private final Log log = Log.getInstance(getClass());
+
+    @Test
+    public void testLogToFile() throws IOException {
+        final File logFile = File.createTempFile(getClass().getSimpleName(), ".tmp");
```

Comment:
Wrap this in a` try-with-resources {} finally{}` so 
1.  The print stream is closed
2.  Capture the original global logger and reset it in the finally.